scripts/release/proposal.js

```diff
@@ -0,0 +1,128 @@
+'use strict'
+
+/* eslint-disable no-console */
+
+// TODO: Support major versions.
+
+const { execSync } = require('child_process')
+const fs = require('fs')
+const path = require('path')
+
+// Helpers for colored output.
+const log = msg => console.log(msg)
+const success = msg => console.log(`\x1b[32m${msg}\x1b[0m`)
+const error = msg => console.log(`\x1b[31m${msg}\x1b[0m`)
+const whisper = msg => console.log(`\x1b[90m${msg}\x1b[0m`)
+
+const currentBranch = capture('git rev-parse --abbrev-ref HEAD')
+const releaseLine = process.argv[2]
+
+// Validate release line argument.
+if (!releaseLine || releaseLine === 'help' || releaseLine === '--help') {
+  log('Usage: node scripts/release/prepare <release-line> [release-type]')
+  process.exit(0)
```

Comment:
```suggestion
  process.exit(1)
```

Since this is also not a successful exit

Reply:
It's only not successful in the first case but not the other 2.

Reply:
I'd also argue that it's successful for the first case as well if the default command is `help`.

Reply:
Eh, that's fine.